images/macos/toolsets/toolset-15.json

```diff
@@ -184,7 +184,8 @@
                     "versions": [
                         "3.0.*",
                         "3.1.*",
-                        "3.2.*"
+                        "3.2.*",
+                        "3.3.*"
```

Comment:
We don't need it since later we will install Ruby 3.3 as a default version. Raw 107 of this file.